tests/by-util/test_hashsum.rs

```diff
@@ -350,11 +401,11 @@ fn test_check_md5sum_reverse_bsd() {
         }
         at.write(
             "check.md5sum",
-            "60b725f10c9c85c70d97880dfe8191b3 a\n\
-             bf35d7536c785cf06730d5a40301eba2  b\n\
-             f5b61709718c1ecf8db1aea8547d4698 *c\n\
-             b064a020db8018f18ff5ae367d01b212 dd\n\
-             d784fa8b6d98d27699781bd9a7cf19f0  ",
+            "60b725f10c9c85c70d97880dfe8191b3  a\n\
+             bf35d7536c785cf06730d5a40301eba2   b\n\
+             f5b61709718c1ecf8db1aea8547d4698  *c\n\
+             b064a020db8018f18ff5ae367d01b212  dd\n\
+             d784fa8b6d98d27699781bd9a7cf19f0   ",
```

Comment:
Please add this as a new test, because the old file should *also* pass!

Reply:
sure